Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/BatchContext.cs

```diff
@@ -1,34 +1,34 @@
 ﻿using Renci.SshNet.Sftp;
 
-namespace Frends.SFTP.DownloadFiles.Definitions
+namespace Frends.SFTP.DownloadFiles.Definitions;
+
+/// <summary>
+/// BatchContext class for creating collection of Task parameters
+/// </summary>
+internal class BatchContext
 {
-    /// <summary>
-    /// BatchContext class for creating collection of Task parameters
-    /// </summary>
-    internal class BatchContext
-    {
-        public Info Info { get; set; }
+    public Info Info { get; set; }
 
-        public string TempWorkDir { get; set; }
+    public string TempWorkDir { get; set; }
```

Comment:
You are not using this property anywhere, so it can be removed.

Reply:
Removed